tensorflow/tools/ci_build/Dockerfile.rocm

```diff
@@ -58,23 +58,27 @@ RUN apt-get update --allow-insecure-repositories && DEBIAN_FRONTEND=noninteracti
 RUN apt-get update --allow-insecure-repositories && \
     DEBIAN_FRONTEND=noninteractive apt-get install -y --allow-unauthenticated \
     rocm-dev rocm-libs rocm-utils rocm-cmake \
+    rocfft miopen-hip miopengemm rocblas hipblas rocrand \
     rocm-profiler cxlactivitylogger && \
     apt-get clean && \
     rm -rf /var/lib/apt/lists/*
-# rocfft miopen-hip miopengemm rocblas hipblas rocrand \
 
 # Build HCC from source, cleanup the default HCC to avoid issue
 #RUN rm -rf /opt/rocm/hcc-1.0 && rm -rf /opt/rocm/lib/*.bc
-#RUN cd $HOME && git clone --recursive -b clang_tot_upgrade https://github.com/RadeonOpenCompute/hcc.git
+#RUN cd $HOME && git clone --recursive -b <> https://github.com/RadeonOpenCompute/hcc.git
```

Comment:
@deven-amd do we need `<>`?

Reply:
the <> is just a place holder for future.
did not want to leave the clanf_tot_upgrade in there, because the reader might think it was somehow signficant
did not want to delete the comment in case we need to go the dropbox route again in the future

Reply:
probably we want to describe it better?
I'll take the PR in its current form now, please submit another one to address the comments.